insonmnia/hardware/gpu/device_test.go

```diff
@@ -0,0 +1,39 @@
+package gpu
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/sonm-io/core/proto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var deviceIdsTable = []struct {
```

Comment:
sorry guys, but i think it worse solution of all to do this.
its not be determined properly only by yours list of vendor number.
One of solution which can i offer you - use [opencl library](https://github.com/KhronosGroup/OpenCL-Headers/blob/master/opencl11/CL/cl.h#L473)  
[ps](https://www.kryptonitelock.com/content/dam/kryt-us/menu-svgs/bicycle.svg)